src/app/pages/patidex/patidex.page.component.html

```diff
@@ -1 +1 @@
-<p>patidex works!</p>
```

Comment:
# **The name need to change for something like (patidex-list or patidex-component)**

Reply:
😾😾😾😾😾😾😾😾😾

Reply:
[https://media.gqmagazine.fr/photos/5b991b5791715b0011253e54/16:9/w_1280,c_limit/un_emoji_pour_les______tous_3480.jpeg](url)

Reply:
Or not.